pulpcore/cli/core/context.py

```diff
@@ -38,13 +38,20 @@ class PulpArtifactContext(PulpEntityContext):
     HREF = "artifact_href"
     LIST_ID = "artifacts_list"
     READ_ID = "artifacts_read"
+    CREATE_ID = "artifacts_create"
+
+    def _create(self, file: IO[bytes]) -> Any:
```

Comment:
```suggestion
    def create(self, file: IO[bytes]) -> Any:
```
Why should we hide that interface?

Reply:
Mypy gets upset that the signature doesn't match [`create()` from `PulpEntityContext`](https://github.com/pulp/pulp-cli/blob/8d164d7a60321e970ff710cb0e8ebd71e4618a55/pulpcore/cli/common/context.py#L439-L444).

I could change the name of this method? Or I could use the signature from `create()` in `PulpEntityContext` and pull the file from `body` but that seems a bit awkward.

Reply:
I would extend the `PulpEntityContext` create with `uploads` the same way as update has it.